selfservice/flow/settings/handler_test.go

```diff
@@ -181,7 +200,7 @@ func TestHandler(t *testing.T) {
 
 		t.Run("description=init a flow as SPA", func(t *testing.T) {
 			t.Run("description=success", func(t *testing.T) {
-				user1 := testhelpers.NewHTTPClientWithArbitrarySessionToken(t, reg)
+				user1 := testhelpers.NewDefaultClientWithSessionCookie(t, reg)
```

Comment:
Should be using `NewDefaultClientWithSessionToken`

Reply:
Actually, this might've been incorrect in the original test- the SPA tests generally use the cookie clients, so I think this change is okay.